frontend/src/components/nodes/__test__/NodeRow.test.tsx

```diff
@@ -12,15 +12,17 @@ describe("<NodeRow />", () => {
             moniker: "",
             accountId: "",
             nodeId: "",
-            lastSeen: 12345,
-            lastHeight: 12345,
-            lastHash: null,
-            signature: null,
-            agentName: null,
-            agentVersion: null,
-            agentBuild: null,
-            peerCount: null,
-            isValidator: null
+            lastSeen: Date.now(),
```

Comment:
I expect it to make the test flaky (failing randomly). Does it work?

Reply:
I never see it flaky...